indico/modules/events/abstracts/forms.py

```diff
@@ -514,6 +515,33 @@ def __init__(self, *args, **kwargs):
         super(NoTrackMixin, self).__init__(*args, **kwargs)
 
 
+class SelectMultipleWidget(Select):
+    """
+    Renders a select field which supports <optgroup>
+    """
+
+    def __call__(self, field, **kwargs):
+        kwargs.setdefault('id', field.id)
+        if self.multiple:
+            kwargs['multiple'] = True
+        html = ['<select %s>' % html_params(name=field.name, **kwargs)]
+        group_iter = groupby(field.iter_choices(), lambda (_val, _label, _selected, group): group)
+        for group, tracks in group_iter:
+            print group, tracks
+            if group:
+                html.append('<optgroup %s>' % html_params(label=group.title))
```

Comment:
use `get_label` or add a new `get_group_label` kwarg to the field

Reply:
Why is this neceessary here?

Reply:
You already have `get_group` to make this field pretty generic, so I think it'd make sense to make this part generic as well instead of hardcoding a title attribute

Reply:
I see what you mean